### ISSUE TITLE
`EXTENSION_FUNCTION_WITH_CLASS` false positive with external interfaces

### ISSUE THREAD

**Issue by akuleshov7** [saveourtool/diktat#1586]
```kotlin
external interface A: B {
}

fun A.extensionMethod() {}
```

In this example we are not able to move extensionMethod to interface
